chia/types/blockchain_format/coin.py

```diff
@@ -22,7 +22,11 @@ class Coin(Streamable):
     def name(self) -> bytes32:
         # This does not use streamable format for serialization. Look at the __bytes__ method that is being overridden:
         # The amount is serialized using CLVM serialization.
-        return self.get_hash()
+
```

Comment:
Hello, I'm currently trying to understand the code a bit. Thanks for documentation!
I'd remove this line breaking the doc (which should probably be a docstring, but okay it wasn't before.) I'd also start the sentence with a capital a N and the last sentence is sort of not really a full sentence.